daffodil-runtime1/src/main/scala/org/apache/daffodil/externalvars/ExternalVariablesLoader.scala

```diff
@@ -53,17 +54,9 @@ object ExternalVariablesLoader {
     varsKVP
   }
 
-  def getVariables(vars: Map[String, String]): Seq[Binding] = {
```

Comment:
Loving this IDEA quick "Find Usages" feature. So easy to find methods that are not in use anywhere!